run-in-retro-term.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# cool-retro-term support for tetris.sh
+# Author: Rojen Zaman <rojen@riseup.net>
+# License: WTFPL
+
+stdout="/dev/null"                                              # default stdout, optional verbose mode is /dev/stdout
+script_dir=$(dirname $(readlink -f $0))                         # script dir, it is require for cool-retro-term
+
+# usage
+usage() {
+    printf "%s\n" \
+        "usage : $0 [-h] [-v]" \
+        " -h   help" \
+        " -v   verbose"
+    exit 0
+}
+
+# check cool-retro-term and xdg-open commands
+check_commands() {
+    local message=()
+    command -v xdg-open &>$stdout || message+="xdg-open could not be found, please install it"
+    command -v cool-retro-term &>$stdout || message+="cool-retro-term could not be found, please install it"
+    [ ${#message} == 0 ] && return 0
+    printf "%s\n" "${message[@]}"
+    echo "Are you sure you are using GNU operating system ;-)?"
+    exit 1
+}
+
+# music loop
+run_music() {
+    while :
+    do                                                          # play music with xdg-open
+        xdg-open "$script_dir/media/tetris-theme.ogg" &>$stdout
+        sleep 85                                                # sleep 85, 85 is total of music time.
```

Comment:
Do you know if there is any way to detect completion of the playback? Generally hardcoded sleeps are not good and can cause issues. Probably not a big deal here, but just curious if better alternative is possible?

Reply:
I searched but couldn't find it, you know?

Reply:
Unfortunately I don't know from the top of my head :(. I did a quick search but couldn't find anything useful. Quick experiment with xdg-open also didn't revealed any low hanging fruites. Oh well, I guess we will use sleep for now.